### DISCUSSION TITLE
What is changing?

### DISCUSSION THREAD

**Discussion by rsmith02ct** [Purfview/whisper-standalone-win#42] [Q&A] [answered]
There have been many version numbers over the past several weeks but when I try to see what's changing by clicking on commits it appears to be the readme? 

Can it be made clearer what changes each incremental version have? I don't need to re-download it if it's just the readme.

**Comment by Purfview** [comment, marked as answer]
This repo doesn't contain code so you can't see any "commits" here.

Major version numbers correspond to revisions in relevant repo, e.g., for whisper-faster.exe there:
https://github.com/guillaumekln/faster-whisper/commits/master

Minor version numbers indicate my own cli tweaks, bugfixes or stuff people ask to implement at Discussions or Issues.
Sometimes I regret not keeping log of cli changes, but that's for me harder to trace some regression...

**Comment by rsmith02ct** [comment]
Thank you, that link is helpful as there have hardly been version changes
on the faster-whisper side.

If you can include some oinfo it helps me understand what changed. I'll
prioritize new versions (whole numbers?)
Overall it's working very well and thanks for continuing to work on it.

**Reply by Purfview** [reply]
In a release notes I add some info about new feature[s] and a link to the last included revision from master repo.

You should always get the latest version, they do something better or generally better than previous version.
